packages/nimble-components/src/rich-text/models/tests/markdown-serializer.spec.ts

```diff
@@ -51,11 +55,31 @@ describe('Markdown serializer', () => {
                 html: '<em>Italics</em>',
                 markdown: '*Italics*'
             },
+            {
+                name: 'Link',
+                html: '<p><a href="#">Link</a></p>',
```

Comment:
Consider leaving a comment stating why we have `a` tag in serialization. I guess tiptap will convert this to `nimble-anchor` using `renderHTML`

Reply:
Added a comment here